raitools/README.md

```diff
@@ -0,0 +1,16 @@
+# Responsible AI Tools SDK for Python
+
+### This package has been tested with Python 3.6, 3.7 and 3.8
+
+The Responsible AI Tools SDK enables users to analyze their machine learning models in one API. Users will be able to analyze errors, explain the most important features, validate fairness, compute counterfactuals and run causal analysis in a single API.
+
+Highlights of the package include:
+
+- `fairness.add()` allows users to run Fairlearn to assess model fairness
+- `explainer.add()` allows users to explain their model
+- `counterfactuals.add()` allows users to compute counterfactuals
+- `error_analysis.add()` allows users to run error analysis
+- `causal.add()` allows users to run causal analysis
```

Comment:
Weren't these suppose to to fairness_manager, explanation_manager and so on.

Reply:
see this internal thread https://teams.microsoft.com/l/message/19:4df5f33d000a4ec98cfe6e17122ce022@thread.tacv2/1617285188141?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=fb4c3c0e-a518-43fa-9b92-96b7bdc852f7&parentMessageId=1617283381204&teamName=AzureML%20Responsible%20AI&channelName=RAI%20Engineering&createdTime=1617285188141
pasting image below if it helps:
![image](https://user-images.githubusercontent.com/24683184/114412908-89327480-9b7b-11eb-8524-46b0a81cad8f.png)
note we use fairness directly, not fairness_manager now